README.md

```diff
@@ -1,16 +1,31 @@
 # national-flow-observations
-This repository pulls national flow data from NWIS. The pipeline pushes to a shared cache in an S3 bucket called [ds-pipeline-national-flow-observations](https://s3.console.aws.amazon.com/s3/buckets/ds-pipeline-national-flow-observations/?region=us-west-2&tab=overview).
+This repository pulls national flow data from NWIS for 50 states and all territories. The pipeline pushes to a shared cache in an S3 bucket called [ds-pipeline-national-flow-observations](https://s3.console.aws.amazon.com/s3/buckets/ds-pipeline-national-flow-observations/?region=us-west-2&tab=overview).
```

Comment:
@wdwatkins what do you think of the doc saying this instead?

```suggestion
This repository pulls national flow data from NWIS for 50 states and all territories. The pipeline pushes to a shared cache in a USGS-internal S3 bucket called [ds-pipeline-national-flow-observations](https://s3.console.aws.amazon.com/s3/buckets/ds-pipeline-national-flow-observations/?region=us-west-2&tab=overview).
```

Reply:
That's fine, maybe just add in parentheses that you'll need to log to the console first for that link to work.